src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/SystemIPAddressInformation.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.NetworkInformation
+{
+    // This is the main addressinformation class that contains the ipaddress
+    // and other properties.
+    internal class SystemIPAddressInformation : IPAddressInformation
+    {
+        private IPAddress _address;
+        internal bool Transient = false;
+        internal bool DnsEligible = true;
```

Comment:
_transient and _dnsEligible?

Also, they're being set in the ctor... they don't need to be initialized to true here.